src/components/studies/StudyBuilder.tsx

```diff
@@ -138,6 +139,23 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
   const setData = (builderInfo: StudyInfoData) => {
     studyDataUpdateFn({ type: 'SET_ALL', payload: builderInfo })
   }
+
+  useEffect(() => {
```

Comment:
This should not be in the state  as you can derive this value from what you already have in the state. Just make this a function that returns a boolean and call this function where you reference that state var.
also something like this seems simpler

```
return sessions && sessions.length && !sessions.find(session=> !session.assessments || session.assessments.length === 0)
```
we can simplify it farther by using lodash method isEmpty since we already have lodash (_) included in this file.
 `return !_.isEmpty(sessions) &&  !sessions.find(session=>_.isEmpty(session.assesments))`